app/javascript/packages/document-capture/components/document-capture.tsx

```diff
@@ -144,7 +132,7 @@ function DocumentCapture({ isAsyncForm = false, onStepChange = () => {} }: Docum
                 : ReviewIssuesStep,
             title: t('errors.doc_auth.throttled_heading'),
           },
-        ] as FormStep[]
+        ] as unknown as FormStep[]
```

Comment:
this seems suspect to me

Reply:
Not 100% clear on what's going on here, but it looks like ReviewIssuesStep needs to be augmented with those `remainingAttempts`, `isFailedResult`, `captureHints`, and `pii` props to match the `FormStep` type (that's what `withProps` is doing). Unclear on what those values would be if `submissionError` is not an instance of `UploadFormEntriesError`--probably more digging needed?

Reply:
Cleaned up. The `unknown` conversion is no more.